lib/Dancer2/Manual.pod

```diff
@@ -3090,7 +3075,7 @@ Or to request a specific layout, of course:
     };
 
 Some tokens are automatically added to your template (C<perl_version>,
-C<dancer_version>, C<settings>, C<request>, C<params>, C<vars> and, if you
+C<dancer_version>, C<settings>, C<request>, C<query_parameters>, C<body_parameters>, C<route_parameters>, C<vars> and, if you
 have sessions enabled, C<session>). Check L<Dancer2::Core::Role::Template>
```

Comment:
@SysPete You were correct, within a template `route_parameters`, `body_parameters`, and `query_parameters` are not exposed. You still must reference them from `params('route')`, etc... I pushed a commit that reverts this change. 

It seems like `route_parameters`, `body_parameters`, and `query_parameters` are now the recommended way to access this data, should I file an issue to add expose them within the template engine similar to `params`?